### PR TITLE
[2.8] MOD-8221: backport info fields (for metrics) to 2.8

### PR DESCRIPTION
**Pull request by oshadmi**
Backport https://github.com/RediSearch/RediSearch/pull/4640 https://github.com/RediSearch/RediSearch/pull/4590 https://github.com/RediSearch/RediSearch/pull/4561
And https://github.com/RediSearch/RediSearch/pull/5146
to 2.8

MOD-8221
(adding info fields)

### CODE REVIEW

File: tests/pytests/test_info.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from common import *
+import time
```
Comment (raz-mon) [inline tests/pytests/test_info.py:2]: Needed?

**Review by raz-mon — COMMENTED** [review]
Looks good 👍
One comment

File: tests/pytests/test_info.py (hunk 1 above)
Comment (oshadmi) [reply tests/pytests/test_info.py]: Removed.

**Review by raz-mon — APPROVED** [review]

**Review by raz-mon — APPROVED** [review]
We probably need to add the changes to `rocky_linux_8.sh` to other versions as well

**Pull request merged**